src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/AppProvisioningTool.cs

```diff
@@ -50,11 +49,19 @@ public AppProvisioningTool(string commandName, ProvisioningToolOptions provision
                 var csProjfiles = Directory.EnumerateFiles(ProvisioningToolOptions.ProjectPath, "*.csproj");
                 if (csProjfiles.Any())
                 {
+                    if (csProjfiles.Count() > 1)
+                    {
+                        string errorMssg = "Specify one .csproj file for the --project-path";
+                        ConsoleLogger.LogJsonMessage(new JsonResponse(CommandName, State.Fail, errorMssg));
```

Comment:
I think this is the only place I see LogJsonMessage and LogMessage back to back; I think that the properties of the ConsoleLogger should be addressing that.  What is special about this use?

Reply:
You should see them back to back through this file. The only places I'm doing this is if the message/content is exactly the same. As the tool will mature, I expect the console output for json and CLI to divert.